src/stash/anchor.rs

```diff
@@ -211,7 +212,7 @@ impl Anchor {
             let mut container = TxContainer {
                 tx: tx.clone(),
                 fee,
-                protocol_factor: vout as u32,
+                protocol_factor: protocol_factor as u32,
```

Comment:
According to my suggested change above, there would be no need to cast `protocol_factor` here. So, it should be written only as:

```suggestion
                protocol_factor: protocol_factor,
```